indico/htdocs/sass/partials/_jquery-indico-taglist.scss

```diff
@@ -20,16 +20,16 @@
     @include border-all();
 
     height: auto;
-    min-height: 50px;
     cursor: text;
-    padding: 5px;
+    padding: 0.3rem;
```

Comment:
You convince me, but maybe we should settle on a strategy for this before changing `px` to `rem` everywhere.